src/lib/utils/data-handlers/resources/resource.ts

```diff
@@ -36,7 +36,7 @@ interface ResourceContentInfoForPath {
     version?: number;
 }
 
-function resourceContentApiPath(resourceContent: ResourceContentInfoForPath) {
+export function resourceContentApiPath(resourceContent: ResourceContentInfoForPath) {
```

Comment:
Looks like this change was undone from my cleanup PR, should be able to revert this